Assets/ConsentManagementProvider/Scripts/facade/CMP.cs

```diff
@@ -51,6 +60,12 @@ public static void Initialize(List<SpCampaign> spCampaigns, int accountId, strin
         
         public static void LoadMessage(string authId = null)
         {
+            if (IsEditor)
```

Comment:
What if we extract this logic to our own logger class? So you can always call the log method and inside the method you'd have that conditional `if (IsEditor)` .

Reply:
Warning message is a bit different in each and every call so I'd rather leave it as it is

Reply:
It's just a matter of receiving the message as a parameter :) 

```csharp
interface Logger
{
	void log(string message);
}

class EditorLogger: Logger
{
	public void log(string message) {}
}

class NonEditorLogger: Logger
{
	public void log(string message)
	{
        	Debug.LogWarning(message);
	}
}

// somewhere in the code
public Logger logger = isEditor ? new EditorLogger() : new NonEditorLogger();
```